.github/workflows/build-future.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       arch: amd64
       tag: ${{ inputs.tag-prefix }}future-amd64
       protocol_version_default: 23
-      xdr_ref: v23.0.0-rc.2
-      core_ref: v23.0.0rc4
+      xdr_ref: v23.0.0
+      core_ref: v23.0.0
```

Comment:
You need to make the same change in the arm64 change below for both files.

Reply:
Done.